libminifi/src/core/state/Value.cpp

```diff
@@ -34,25 +34,29 @@ const std::type_index Value::BOOL_TYPE = std::type_index(typeid(bool));
 const std::type_index Value::DOUBLE_TYPE = std::type_index(typeid(double));
 const std::type_index Value::STRING_TYPE = std::type_index(typeid(std::string));
 
-void hashNode(const SerializedResponseNode& node, SHA512_CTX& ctx) {
-  SHA512_Update(&ctx, node.name.c_str(), node.name.length());
+void hashNode(const SerializedResponseNode& node, EVP_MD_CTX* ctx) {
```

Comment:
If ctx is expected to be valid/not-null, then keeping it as a reference would document this on the interface.

Reply:
Updated in 5ebb285e42f5ea89391d5b126bcbecb2b27c77d9